server/src/main/java/org/elasticsearch/common/blobstore/DeleteResult.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.common.blobstore;
+
+/**
+ * The result of deleting multiple blobs from a {@link BlobStore}.
+ */
+public final class DeleteResult {
+
+    public static final DeleteResult ZERO = new DeleteResult(0, 0);
+
+    private final long blobsDeleted;
```

Comment:
nit: this could be just `blobs` & `bytes`

Reply:
Hmm, I'd rather not, then I have to use `this` in `add` :D